action.yml

```diff
@@ -6,7 +6,21 @@ inputs:
     description: "Working directory relative to the root directory."
     required: false
     default: "."
+  format:
+    description: "Don't write the files back, just return the status."
+    required: false
+    default: "false"
+  fail_on_error:
```

Comment:
What's moved?

If option was related, add more info in description for `format`.

Reply:
https://github.com/reviewdog/action-black/blob/da08ab17ca7406548245f2d3940f924ad9ff5056/entrypoint.sh#L75-L78

(note)